src/Store/VimStoreConnector.re

```diff
@@ -79,7 +79,7 @@ let start =
 
   let _ =
     Vim.onDirectoryChanged(newDir =>
-      dispatch(Model.Actions.OpenExplorer(newDir))
+      dispatch(Model.Actions.VimDirectoryChanged(newDir))
```

Comment:
This looks like a good example of renaming from the 'expected action' to a describing an event that has occurred 👍

Reply:
Yeah, this is a good example of something that has a high likelihood of causing a bug in the future. If we want something more to happen when we click the explorer button in the sidebar for example, it seems natural to put that in the `OpenExplorer` action. But you probably do not want that same thing to happen when you `:cd`.